### PR TITLE
atx.intro1.jimenez.christopher

### PR DESCRIPTION
**Pull request by cjimenez113**

### CODE REVIEW

**Comment by e-zambrano** [comment]
@cjimenez113 - after fighting with GitHub you are finally able to commit changes to your pull request. Everything looks good, excellent work here

File: 03LessonThree/homework/git-scm.html [hunk 1]
```diff
@@ -3,58 +3,61 @@
...
+        <nav>
             <span><a href="#">Home</a></span>
             <span><a href="#">How does it work</a></span>
             <span><a href="#">Contact Us</a></span>
-        </div>
-    </div>
-    <div id="main">
-        <div id="section">
-            <img src="http://git-scm.com//images/logo@2x.png" alt="git logo"/>
+
```
Comment (e-zambrano) [inline 03LessonThree/homework/git-scm.html:15]: where is the `</nav>`?

File: 03LessonThree/homework/git-scm.html [hunk 2]
```diff
@@ -3,58 +3,61 @@
...
             </div>
             <div>
-            <h4>Node.js®</h4>
-            <p>It is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.</p>
-            <img src="http://nodejs-cloud.com/img/128px/nodejs.png">
-            </div>
+                <h4>Node.js®</h4>
+                <p>It is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.</p>
+                <img src="http://nodejs-cloud.com/img/128px/nodejs.png" />
+        </aside>
```
Comment (e-zambrano) [inline 03LessonThree/homework/git-scm.html:55]: i think this `</aside>` and the `</div>` below are in the wrong place. if you fix that i believe your homework will work

File: 05LessonFive/Checkpoint1/css/style.css [hunk 3]
```diff
@@ -1,43 +1,31 @@
...
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
-}
-
-.left h1 {
-    text-align: center;
+h1{
+     color:#252b3b;
+    font-size:1.9em
```
Comment (e-zambrano) [inline 05LessonFive/Checkpoint1/css/style.css]: missing a `;`at end of the declaration

File: 06LessonSix/in-class-project/css/intel-style.css [hunk 4]
```diff
@@ -1,29 +1,41 @@
 body{
-    width:80%;
-    margin:auto;
+    font-family:'Arial',
```
Comment (e-zambrano) [inline 06LessonSix/in-class-project/css/intel-style.css]: It's not a comma what goes after a declaration.
What it is that it goes?

File: 06LessonSix/in-class-project/css/intel-style.css [hunk 5]
```diff
@@ -1,29 +1,41 @@
 body{
-    width:80%;
-    margin:auto;
+    font-family:'Arial',
+    sans-serif;
 }
-header{
+Header,main,footer{
```
Comment (e-zambrano) [inline 06LessonSix/in-class-project/css/intel-style.css]: I don't think `Header` was supposed to be capitalize

**Comment by e-zambrano** [comment]
@cjimenez113 - Please attend to the Sunday's study sessions to fix your unfinished projects.

File: 09LessonNine/in-class-project/css/style.css [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+main {
+  background-image:url("../img/featured-image-4.jpg");
+  background-repeat: no-repeat;
+  background-position:center;
+  background-size:cover;
+  height:350px;
+  width:100%;
+  text-align: center;
+  padding-top:100px;
+  color:white;
```
Comment (fasteddiegarcia) [inline 09LessonNine/in-class-project/css/style.css]: You are missing the code to darken the background-images, Chris. Look in Slack in the atx-intro channel for the code.

File: 09LessonNine/in-class-project/css/style.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+h1{
+  padding-top:75px;
+}
+.leftbox, .rightbox {
+  float:left;
+  color:white;
+  height:400px;
+  width:50%;
+  text-align:center;
+  margin-top:20px;
```
Comment (fasteddiegarcia) [inline 09LessonNine/in-class-project/css/style.css]: I'd like you to finish this site out. Add the appropriate margin to these divs so this page looks exactly like the example, please.

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 8]
```diff
@@ -1,44 +1,83 @@
...
+  margin-bottom: 15px;
+  background-color: #f2f2f2;
+  padding: 10px 10px;
+}
+.footer{
+  margin: 50px auto;
+  width: 50%;
+  border: 1px solid;
+  border-color: whitesmoke;
+  padding: 10px 49px;
```
Comment (fasteddiegarcia) [inline 08LessonEight/in-class-project/css/homework-style.css:69]: The footer only has top and bottom border, not all the way around.

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 9]
```diff
@@ -1,44 +1,83 @@
...
+}
+.footerbox{
+  float: left;
+}
+.footerbox1{
```
Comment (fasteddiegarcia) [inline 08LessonEight/in-class-project/css/homework-style.css:74]: There are "border-right"s on the divs inside the footer.

File: 08LessonEight/homework/exercise-06.html [hunk 10]
```diff
@@ -7,61 +7,25 @@
...
+    <p>This is a good way to devliver stuff</p>
+    <a href="#">Read More</a>
+    </div>
+    <div class="train">
+        <img src="img/train.jpg">
+            <h3>Safe & Secured Delivery</h3>
+        <p>This is a good way to devliver stuff</p>
+        <a href="#">Read More</a>
+    </div>
+    </div>
```
Comment (fasteddiegarcia) [inline 08LessonEight/homework/exercise-06.html:28]: Great job!

File: 08LessonEight/homework/exercise-06.html [hunk 11]
```diff
@@ -7,61 +7,25 @@
...
-    </footer>
+    <div class="bigtruck">
+        <img src="img/truck.jpg" />
+        <h3>Safe & Secured Delivery</h3>
+        <p>This is one of the best ways to move your stuff</p>
+        <a href="#">Read move</a>
+    </div>
+    <div class="airplane" />
+    <img src="img/plane.jpg" />
+      <h3>Safe & Secured Delivery</h3>
```
Comment (fasteddiegarcia) [inline 08LessonEight/homework/exercise-06.html:18]: I'd like you to beautify your code every 10 mins. So you, a human, can read it easier and therefore organize your thoughts easier.

File: 11LessonEleven/in-class-project/css/style.css [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+  float:left;
+}
+  .clearfix:after {
+   content:"";
+   display:block;
+   clear:both;
+ }
+
+}
+@media max-width(768px){
```
Comment (e-zambrano) [inline 11LessonEleven/in-class-project/css/style.css:27]: You are writing the `media query` incorrectly. That's why it's not working.
Review the lesson to get this fixed.

**Comment by e-zambrano** [comment]
@cjimenez113 - You are doing excellent

File: 02LessonTwo/homework/css/style.css [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+#header {
+	z-index: 1;
+	position: fixed;
+	width: 97.5%;
+	margin-top: -20px;
+	height: 60px;
+	background-color: #668284;
+	margin-bottom: 10px;
+}
+
```
Comment (zacacollier) [inline 02LessonTwo/homework/css/style.css]: good work!

File: 13LessonThirteen/homework/css/style.css [hunk 14]
```diff
@@ -0,0 +1,45 @@
+header {
+    background-image: url("http://bone.themanoid.com/images/office/03.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 920px;
+}
+h1 {
+    text-align: center;
+    padding-top: 400px;
```
Comment (zacacollier) [inline 13LessonThirteen/homework/css/style.css]: Too much padding !

File: 13LessonThirteen/homework/index.html [hunk 15]
```diff
@@ -1,10 +1,69 @@
...
+        </div>
+        <div class="col-lg-4">
+            <img class="right img-responsive" src="http://bone.themanoid.com/images/thumbs/03.png">
+        </div>
+    </container>
+    <footer class="grey visible-xs">
+        <div class="container">
+            <div class="row">
+                <div class="col-xs-12 text-center">
+                        <h2>A fantastic Price</h2>
```
Comment (zacacollier) [inline 13LessonThirteen/homework/index.html]: Nicely done!

**Pull request closed without merging**